README.md

```diff
@@ -48,6 +48,7 @@ Check out our getting started tutorial on launching your first Kubernetes cluste
   * [Backup Kubernetes resources](/Documentation/kubernetes-on-aws-backup.md)
   * [Restore Kubernetes resources](/contrib/cluster-backup/README.md)
   * [Journald logging to AWS CloudWatch](/Documentation/kubernetes-on-aws-journald-cloudwatch-logs.md)
+    * [kube-aws up/update feedback](/Documentation/kubernetes-on-aws-journald-cloudwatch-logs.md)
```

Comment:
Ah, good catch @ankon!
Perhaps `/Documentation/kubernetes-on-aws-journald-cloudwatch-logs.md#kube-aws-up-update-feedback` would work. Also see [this SO question](https://stackoverflow.com/a/38507669) for the anchor naming rule in GH flavored markdown.